src/main/resources/config/application-default.yml

```diff
@@ -2,10 +2,19 @@ spring:
     profiles:
         active: dev
     datasource:
-        platform: mysql
+        platform: postgres
```

Comment:
A copy-paste error from me. It seems that the correct platform is `postgresql`, see https://docs.spring.io/spring-boot/docs/current/reference/html/howto-database-initialization.html#howto-initialize-a-database-using-spring-jdbc.